pittapi/gym.py

```diff
@@ -0,0 +1,82 @@
+"""
+The Pitt API, to access workable data of the University of Pittsburgh
+Copyright (C) 2015 Ritwik Gupta
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+"""
+
+from bs4 import BeautifulSoup
+import requests
+from typing import NamedTuple, List
+
+GYM_URL = "https://connect2concepts.com/connect2/?type=bar&key=17c2cbcb-ec92-4178-a5f5-c4860330aea0"
+
+
+class Gym(NamedTuple):
+    name: str
+    date: str
+    count: int
+    percentage: int
+
+
+def get_all_gyms_info() -> List[Gym]:
```

Comment:
Let's use the built-in `list` annotation rather than the imported `List`. Since this built-in annotation wasn't available until 3.10 (and we're trying to support 3.9), make sure to import `annotations` from `__future__`:
```suggestion
from __future__ import annotations

from bs4 import BeautifulSoup
import requests
from typing import NamedTuple

GYM_URL = "https://connect2concepts.com/connect2/?type=bar&key=17c2cbcb-ec92-4178-a5f5-c4860330aea0"


class Gym(NamedTuple):
    name: str
    date: str
    count: int
    percentage: int


def get_all_gyms_info() -> list[Gym]:
```